urdf/package.xml

```diff
@@ -24,12 +24,12 @@
   <build_depend>console_bridge</build_depend>
   <build_depend>urdfdom</build_depend>
   <build_depend>urdfdom_headers</build_depend>
-  <build_depend>TinyXML</build_depend>
+  <build_depend>tinyxml</build_depend>
   <build_depend>tinyxml_vendor</build_depend>
 
   <exec_depend>urdfdom</exec_depend>
-  <exec_depend>urdfdom-headers</exec_depend>
-  <exec_depend>TinyXML</exec_depend>
+  <exec_depend>urdfdom_headers</exec_depend>
```

Comment:
> The same rational should also apply to TinyXML

Can you elaborate on this. I see no reason to create a rosdistro specific "TinyXML" package when the rosdep key named tinyxml exists and is suitable.